### PR TITLE
Update bridge-to-kubernetes-aks.md

### PR DESCRIPTION
**Pull request by elenavillamil**
Replace deprecated bike app with supported todo app

### CODE REVIEW

**Comment by PRMerger3** [comment]
@elenavillamil : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger-2** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 File Change Percent | This PR contains file(s) with more than 20% file change.

File: bridge/articles/bridge-to-kubernetes-aks.md [hunk 1]
```diff
@@ -43,174 +43,109 @@ az aks create \
...
 
-Additionally, you can find the diagnostic logs in the `Bridge to Kubernetes` directory in your development computer's TEMP directory. On Windows 10, that's in `%TEMP%\Bridge to Kubernetes`. On a Mac, the TEMP directory can be found by running `echo $TMPDIR` from a terminal window. On Linux, it is `/tmp/Bridge to Kubernetes`.
+Press play and let the request continue complete transparently.
 
-## Running in isolation mode
+## Clean up
 
-With Bridge to Kubernetes, you can also set up an isolated version the services you're working on, meaning that others who are using the cluster won't be affected by your changes. This isolation mode is accomplished by routing your requests to your copy of each affected service, but routing all other traffic normally. More explanation on how this is done can be found at [How Bridge to Kubernetes Works][btk-overview-routing].
+When you stop the debugger all changes to the cluster are reverted. Note, if you were running with isolation on, the routing manager pod will be left there to spedite the start of teh debugger next time around.
```
Comment (ghogen) [inline bridge/articles/bridge-to-kubernetes-aks.md:145]:
```suggestion
When you stop the debugger all changes to the cluster are reverted. Note, if you were running with isolation on, the routing manager pod will be left there to expedite the start of the debugger the next time around.
```

**Comment by PRMerger16** [comment]
@ghogen : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger10** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 File Change Percent | This PR contains file(s) with more than 20% file change.

**Comment by jborsecnik** [comment]
@ghogen, 

Can you review the proposed changes and let us know when this is good to merge? 

IMPORTANT: When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Comment by qpetraroia** [comment]
Thanks @ghogen, 

Appreciate your help here!

File: bridge/articles/bridge-to-kubernetes-aks.md [hunk 2]
```diff
@@ -43,174 +43,109 @@ az aks create \
...
-
-## Next steps
-
-Learn more about Bridge to Kubernetes at [How Bridge to Kubernetes works][btk-how-it-works].
-
-[azure-kubernetes-service]: /azure/aks/kubernetes-walkthrough
-[azds-cli]: /azure/dev-spaces/how-to/install-dev-spaces#install-the-client-side-tools
-[azds-tmp-dir]: /azure/dev-spaces/troubleshooting#before-you-begin
-[btk-vs-code]: https://marketplace.visualstudio.com/items?itemName=mindaro.mindaro
-[azure-cli]: /cli/azure/install-azure-cli?view=azure-cli-latest&preserve-view=true
```
Comment (ghogen) [inline bridge/articles/bridge-to-kubernetes-aks.md:204]: Some of these links are used in the document, so you can't delete them from this table. For example, [azure-cli] is used in line 24

**Comment by ghogen** [comment]
Moved to the private repo - closing here.

**Pull request closed without merging**